rest/servers/spring/pom.xml

```diff
@@ -146,6 +146,15 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.owasp</groupId>
+                <artifactId>dependency-check-maven</artifactId>
+                <configuration>
+                    <suppressionFiles>
+                        <suppressionFile>${basedir}/owasp-suppressions.xml</suppressionFile>
```

Comment:
Doesn't this mean you could condense all the suppression files above into one?

Reply:
I think that if we wanted to use a single one, we'd have to move it to the root module and use the absolute path to the file, which would force us to use a specific plugin like we did in DDF just to get a property set with the root module's directory.
Using a single suppression file can also cause issues because you could end up suppressing something at the top for one module that shouldn't be suppressed in another.
Also, the suppression files are not all identical.
For those reasons, I decided against using a single file. But I could be convinced otherwise if we think it'd be a better option.

Reply:
Well, it looked like you could still be specific from a single file depending on how you scope the suppression. I just thought it would be easier to keep track of and a little less maintenance in a single file. https://jeremylong.github.io/DependencyCheck/general/suppression.html Unless that doesn't work the way I think it does.

Reply:
We still need to figure out how we're going to deal with vulnerabilities in API repos and how it affects clients using the API. That may drive how we manage those suppression files. I recommend we merge the way it is for now and put up another PR when we know more about how that's going to work.